hoomd/mpcd/ManualVirtualParticleFiller.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2009-2019 The Regents of the University of Michigan
+// This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+// Maintainer: mphoward
+
+/*!
+ * \file mpcd/VirtualParticleFiller.cc
+ * \brief Definition of mpcd::VirtualParticleFiller
+ */
+
+#include "ManualVirtualParticleFiller.h"
+
+mpcd::ManualVirtualParticleFiller::ManualVirtualParticleFiller(std::shared_ptr<mpcd::SystemData> sysdata,
+                                                   Scalar density,
+                                                   unsigned int type,
+                                                   std::shared_ptr<::Variant> T,
+                                                   unsigned int seed)
+    : mpcd::VirtualParticleFiller(sysdata, density, type, T, seed)
```

Comment:
Make sure to default initialize `m_first_tag` and `m_N_fill` here, since they were removed from the base class